src/components/CampaignLocations/CampaignLocationsScreen.tsx

```diff
@@ -127,6 +128,22 @@ export const CampaignLocationsScreen: FunctionComponent<NavigationProps> = ({
     navigateToCampaignLocation,
     navigation
   ]);
+  const authCredentialsWithCampaignName = Object.entries(authCredentials).map(
+    ([key, credentials]) => ({
+      ...credentials,
+      key,
+      name: allCampaignConfigs[key]?.features?.campaignName
+    })
+  );
+
+  const sortedAuthCredentialsWithCampaignName = sortBy(
+    authCredentialsWithCampaignName,
+    [
+      o => {
+        return o.name;
+      }
+    ]
```

Comment:
Can be shortened further:
```suggestion
    "name"
```

Reply:
shortened to [o=>o.name]

Reply:
Just curious, does just putting `"name"` work instead?

Reply:
i tried with name but there were errors, so I used o.name instead.

Reply:
Oh, what was the error?

Reply:
Changed to "name"